src/cell_utils.ts

```diff
@@ -29,10 +29,14 @@ namespace cell_utils {
       return null;
     }
     let out = cell.outputs.toJSON().pop();
+    console.log("out:", out)
```

Comment:
don't need this console.log anymore, delete

Reply:
I removed this and other logging statements with a new commit.